repository-data/development/src/main/resources/hcm-content/hst/configurations/common/workspace/containers/homepage/homepage-blog/bloglist1.yaml

```diff
@@ -0,0 +1,28 @@
+---
+/hst:hst/hst:configurations/common/hst:workspace/hst:containers/homepage/homepage-blog/bloglist2:
+  hst:componentclassname: uk.nhs.digital.common.components.LatestBlogComponent
```

Comment:
Not sure if this file is required?  Is it to enable blog list on frontpage locally?

Reply:
Yes it is to enable blog list on front page locally. Based on my tests it is required as for the the first time commit (when I didn't include this file) blog list didn't work on front page.

Reply:
👍